src/Stratis.Bitcoin/Consensus/PerformanceCounters/ConsensusManager/ConsensusManagerPerformanceSnapshot.cs

```diff
@@ -54,7 +54,7 @@ public ExecutionsCountAndDelay()
 
         public double GetAvgExecutionTimeMs()
         {
-            return Math.Round((this.totalDelayTicks / (double)this.totalExecutionsCount) / 10000.0, 4);
+            return Math.Round(TimeSpan.FromTicks(this.totalDelayTicks).TotalMilliseconds / this.totalExecutionsCount, 4);
```

Comment:
milliseconds is the same as dividing ticks by 1k?

Reply:
Not 1k but 10k, it was 1k until few days ago when I found it was a bug and fixed with 10k but I think getting rid of numbers and using timespan is cleaner (and I saw we did already in other places)